libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilot.kt

```diff
@@ -0,0 +1,92 @@
+package com.mapbox.navigation.copilot
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.copilot.internal.PushStatusObserver
+import com.mapbox.navigation.core.DeveloperMetadataObserver
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationApp
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
+import java.util.concurrent.CopyOnWriteArraySet
+
+/**
+ * Mapbox Copilot is a Navigation SDK component that collects detailed trace files of navigation sessions together with search analytics data.
+ *
+ * Mapbox Copilot collects information that helps customers in two different ways:
```

Comment:
Once "Mapbox Copilot" has been used at the top of the document, you can probably switch to just "Copilot" for the sake of brevity the rest of way.